client-app/core/api/graphql/config/links.ts

```diff
@@ -21,5 +33,16 @@ export const link = from([
   sharedLink,
   // Add conditional links here
   cartLink,
-  httpLink,
+  // Send only subscriptions through web sockets
+  split(
+    ({ query }) => {
+      const definition = getMainDefinition(query);
+      // Disabled because importing this enums cause issues and graphql package will remove them at v17
```

Comment:
create task

Reply:
Done. https://virtocommerce.atlassian.net/browse/VCST-834 Also added to comment in code